src/components/CreateShareStructure/ListShareClass.vue

```diff
@@ -1,14 +1,368 @@
 <template>
-<div></div>
+  <v-card flat id="share-structure">
+
+    <!-- Summary Section -->
+    <div id="share-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="share-summary-header" >
+        <v-icon>mdi-file-tree</v-icon>
+        <label class="share-summary-header-title"><strong> Share Structure</strong></label>
+      </div>
+
+      <!-- Summary Warning -->
+      <div v-if="showErrorSummary" class="share-summary-invalid-message">
+        <span>
+          <v-icon color="#1976d2">mdi-information-outline</v-icon>
```

Comment:
We should avoid using hex colors.
Vuetify includes color names that we can use instead. See https://vuetifyjs.com/en/styles/colors/

In this case, the name of that hex color is "blue darken-2". Please replace it here and elsewhere in the code.

Reply:
done